fdbserver/TestTLogServer.actor.cpp

```diff
@@ -211,11 +212,14 @@ ACTOR Future<Void> TLogTestContext::sendPushMessages(TLogTestContext* pTLogTestC
 		Future<Version> loggingComplete = pTLogTestContext->ls->push(prev, next, prev, prev, toCommit, SpanContext());
 		Version ver = wait(loggingComplete);
 		ASSERT_LE(ver, next);
-		prev++;
-		next++;
+		prev = next;
+		pTLogTestContext->tLogOptions.versions.push_back(next);
+		next += SERVER_KNOBS->VERSIONS_PER_SECOND;
```

Comment:
maybe randomize this to a value within a range?

Reply:
I will add this in the next revision